packages/create-block/lib/templates.js

```diff
@@ -222,15 +225,43 @@ const getDefaultValues = ( pluginTemplate ) => {
 const getPrompts = ( pluginTemplate, keys ) => {
 	const defaultValues = getDefaultValues( pluginTemplate );
 	return keys.map( ( promptName ) => {
+		if ( promptName === 'variant' ) {
+			prompts[ promptName ].choices = pluginTemplate.variants;
+		}
 		return {
 			...prompts[ promptName ],
 			default: defaultValues[ promptName ],
 		};
 	} );
 };
 
+const getTemplateVariantVars = ( variants, variant ) => {
+	const variantTypes = {};
```

Comment:
I wasn't sure what variant types mean at first, maybe we could rename the variable to:
```suggestion
	const variantVars = {};
```

This would align with the function name, what do you think?